INSTALL.md

```diff
@@ -9,10 +9,11 @@ Windows
 Compiling OpenRA requires the following dependencies:
 * [Windows PowerShell >= 4.0](http://microsoft.com/powershell) (included by default in recent Windows 10 versions)
 * [.NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0) (or via Visual Studio)
+* NuGet (included by default in .NET 5 SDK) with online source configured `dotnet nuget add source https://api.nuget.org/v3/index.json`
```

Comment:
From Discord discussion: This step should not be necessary as this is already the default source and therefore should not be listed here.

Reply:
Yet somehow just days after this comment by own fresh install of VS was missing the default nuget source 😢 
So this may not be as redundant as it sounds.

Reply:
It appears VS 2022 does indeed come with no default NuGet source configured.